src/form/Form.jsx

```diff
@@ -1012,6 +1013,16 @@ function Form({
     } else handleRedirect({ metadata });
   };
 
+  useHotkeys('enter', (e) => {
```

Comment:
have u verified the submit functionality still works as expected?

Reply:
Yes, I have tested. I had to add an option to make it behave correctly and consistently.